pkg/controllers/status/cluster_status_controller.go

```diff
@@ -455,8 +456,8 @@ func convertObjectsToNodes(nodeList []runtime.Object) ([]*corev1.Node, error) {
 	nodes := make([]*corev1.Node, 0, len(nodeList))
 	for _, obj := range nodeList {
 		unstructObj := obj.(*unstructured.Unstructured)
-		node := &corev1.Node{}
-		if err := runtime.DefaultUnstructuredConverter.FromUnstructured(unstructObj.UnstructuredContent(), node); err != nil {
+		node, err := helper.ConvertToNode(unstructObj)
```

Comment:
This local variable `unstructObj` can also be removed, directly use `obj.(*unstructured.Unstructured)`.

Reply:
OK

Reply:
Thank you very much.